runtime/cpp/test/fletcher/test.cpp

```diff
@@ -52,10 +55,41 @@ TEST(Platform, EchoPlatform) {
 
   // MMIO:
   ASSERT_TRUE(platform->WriteMMIO(0, 0).ok());
+
```

Comment:
This is also a noteworthy change, it overrides stdin to write some data, because the echo platform normally asks the user for input (which wouldn't work in CI, of course).